docs/sources/reference/components/pyroscope/pyroscope.ebpf.md

```diff
@@ -17,6 +17,12 @@ To use the  `pyroscope.ebpf` component you must run {{< param "PRODUCT_NAME" >}}
 
 You can specify multiple `pyroscope.ebpf` components by giving them different labels, however it's not recommended as it can lead to additional memory and CPU usage.
 
+## Supported languages
+
+This eBPF profiler only collects CPU profiles. Generally natively compiled languages like C/C++, Go, Rust are supported (also see [Troubleshooting]({{< relref "#troubleshooting" >}}) for additional requirements).
```

Comment:
```suggestion
This eBPF profiler only collects CPU profiles. Generally, natively compiled languages like C/C++, Go, and Rust are supported. Refer to [Troubleshooting unknown symbols](#troubleshooting-unknown-symbols) for additional requirements.
```

relrefs are deprecated. We should use inline links or define the links at the bottom of the section or page.

Reply:
Do you have an example by hand?

Reply:
Inline linking is like I suggested here.

Links at the bottom of the section/page is used in most topics - for example in `docs/sources/get-started/configuration-syntax/components.md` (we can use relative links or fully qualified URLs.

Reply:
I see, sorry I wasn't properly reading the suggestion.